Changes

```diff
@@ -565,6 +565,9 @@ Release branch for 4.06:
 - GPR#1308: Only treat pure patterns as inactive
   (Leo White, review by Alain Frisch and Gabriel Scherer)
 
+- MPR#7619, GPR#1387: position of the optional last semi-column not included
+  in the position of the expression (same behavior than for lists)
```

Comment:
same behavior "as"? You should also include an author credit line.

Reply:
Thanks, this is done.